conf/modules.config

```diff
@@ -446,6 +446,7 @@ process {
     }
 
     withName: MOTUS_PROFILE {
+        ext.args = { [ params.save_ncbi_id ? "-p" : "", params.save_read_counts ? "-c" : "", params.save_mgc_counts ?  "-M ${task.ext.prefix}.mgc" : "" ].join(',').replaceAll(','," ") }
```

Comment:
Please could you split this over multiple lines (one per each parameter .e.g like bbduk above)

Reply:
Thanks for your comments, I will update the PR.